File: Basic Coding Problems/02 If - Elses/029 Write a program to check whether a character is uppercase or lowercase alphabet/Java/uppercase_lowercase_alphabet.java
import java.util.Scanner;

public class uppercase_lowercase_alphabet {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a character: ");
        char character = scanner.next().charAt(0);

        if (Character.isUpperCase(character)) {
            System.out.println("The character is uppercase alphabet.");
        } else if (Character.isLowerCase(character)) {
            System.out.println("The character is lowercase alphabet.");
        } else {
            System.out.println("The character is not an alphabet.");
        }
    }
}

File: Basic Coding Problems/02 If - Elses/029 Write a program to check whether a character is uppercase or lowercase alphabet/C/uppercase_lowercase_alphabet.c
#include <stdio.h>

int main() {
    char character;

    printf("Enter a character: ");
    scanf("%c", &character);

    if (character >= 'A' && character <= 'Z') {
        printf("The character is uppercase alphabet.\n");
    } else if (character >= 'a' && character <= 'z') {
        printf("The character is lowercase alphabet.\n");
    } else {
        printf("The character is not an alphabet.\n");
    }

    return 0;
}